index.html

```diff
@@ -1,123 +1,130 @@
               <p>Большинство едет в гранд каньон и толпится там</p>
             </li>
           </ul>
-    <section>
-          <p>Заинтересовались?</p>
-          <p>Укажите предполагаемые даты поездки,  и мы покажем вам лучшие предложения гостиниц в седоне</p>
-          <button type="button">Поиск гостиницы в Седоне</button>
+    <section class="travel">
+          <p class="interested">Заинтересовались?</p>
+          <p class="date">Укажите предполагаемые даты поездки,  и мы покажем вам лучшие предложения гостиниц в седоне</p>
+          <button class="button-find" type="button">Поиск гостиницы в Седоне</button>
           <form action="https://echo.htmlacademy.ru/courses" method="post">
               <p>
                 <label for="check-in-date">Дата заезда</label>
-                <input id="check-in-date" type="text" name="date" placeholder="24 АПРЕЛЯ 2017">
-                <button type="button"><img src="img/calendar.svg" width="21" height="22" alt="календарь"></button>
+                <input class="input-date" id="check-in-date" type="text" name="date" placeholder="24 АПРЕЛЯ 2017">
+                <button class="button-calendar" type="button"><img src="img/calendar.svg" width="21" height="22" alt="календарь"></button>
               </p>
               <p>
                 <label for="departure-date">Дата выезда</label>
-                <input id="departure-date" type="text" name="date" placeholder="4 ИЮЛЯ 2017">
-                <button type="button"><img src="img/calendar.svg" width="21" height="22" alt="календарь"></button>
+                <input class="input-date" id="departure-date" type="text" name="date" placeholder="4 ИЮЛЯ 2017">
+                <button class="button-calendar" type="button"><img src="img/calendar.svg" width="21" height="22" alt="календарь"></button>
               </p>
               <p>
                 <label for="old">Взрослые</label>
-                <button type="button">-</button>
-                <input id="old" type="text" name="quantity" placeholder="2">
-                <button type="button">+</button>
+                <button class="button-minus" type="button">-</button>
+                <input class="input-number" id="old" type="text" name="quantity" placeholder="2">
+                <button class="button-plus" type="button">+</button>
               </p>
               <p>
                 <label for="children">Дети</label>
-                <button type="button">-</button>
-                <input id="children" type="text" name="quantity" placeholder="0">
-                <button type="button">+</button>
+                <button class="button-minus" type="button">-</button>
+                <input class="input-number" id="children" type="text" name="quantity" placeholder="0">
+                <button class="button-plus" type="button">+</button>
               </p>
-              <button type="submit">Найти</button>
+              <button class="find-button"type="submit">Найти</button>
```

Comment:
```suggestion
              <button class="find-button" type="submit">Найти</button>
```
Пропущен пробел между атрибутами.